src/components/Home.jsx

```diff
@@ -13,12 +13,12 @@ import Shortcuts from 'components/Shortcuts'
 
 class Home extends Component {
   render() {
-    const { wrapper } = this.props
+    const { setAppsReady, wrapper } = this.props
     return (
       <Main className="u-flex-grow-1">
         <ScrollToTopOnMount target={wrapper} />
         <Content className="u-flex u-flex-column u-ph-1">
-          <Applications />
+          <Applications onLoad={setAppsReady} />
```

Comment:
I'm not very confortable with the `onLoad` attribute. 

onLoad have a specific meaning: 
>  The load event fires at the end of the document loading process. At this point, all of the objects in the document are in the DOM, and all the images, scripts, links and sub-frames have finished loading. 

(from  The load event fires at the end of the document loading process. At this point, all of the objects in the document are in the DOM, and all the images, scripts, links and sub-frames have finished loading.) 

There this is not true. onLoad is not called when the images are loaded or else. 

It's more about hydrating than loading?

Reply:
I think the idea is more to provide a callback API for when apps are considered "loaded" for good. I'll suggest a better name